src/material/datepicker/datepicker-input.ts

```diff
@@ -180,11 +180,11 @@ export class MatDatepickerInput<D> implements ControlValueAccessor, OnDestroy, V
 
   /** Emits when a `change` event is fired on this `<input>`. */
   @Output() readonly dateChange: EventEmitter<MatDatepickerInputEvent<D>> =
-      new EventEmitter<MatDatepickerInputEvent<D>>();
```

Comment:
please undo the indentation changes in this file

Reply:
done